runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -169,6 +169,33 @@ public void removeTag(int itemId, String tag)
 		}
 	}
 
+	public void renameTag(String oldTag, String newTag)
+	{
+		final String prefix = CONFIG_GROUP + "." + ITEM_KEY_PREFIX;
+		configManager.getConfigurationKeys(prefix).forEach(item ->
+		{
+			int id = Integer.parseInt(item.replace(prefix, ""));
+			renameTag(id, oldTag, newTag);
+		});
+	}
+
+	public void renameTag(int itemId, String oldTag, String newTag)
```

Comment:
This entire function can be replaced by how import sets tags
https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java#L298